activerecord/test/cases/quoting_test.rb

```diff
@@ -24,15 +24,19 @@ def test_quoted_false
       end
 
       def test_quote_column_name
```

Comment:
I'm wondering if we need to test `test_quote_column_name` and `test_quote_table_name` for each database adapters.

Reply:
Would probably be a good idea yes.